crates/schema/src/composed_schema.rs

```diff
@@ -312,11 +454,145 @@ impl ComposedSchema {
                                                 .any(|name| name == &field.node.name.node)
                                         })
                                         .unwrap_or(false);
+
+                                    // Check for incompatible field arguments
+                                    let existing_field = meta_type.fields.get(&field.node.name.node).unwrap();
```

Comment:
Despite the fact that now it shouldn't panic. In future code may change that this unwrap won't succeed. Maybe it's better to wrap this situation into an error, or at least use expect with a message?

Reply:
There are still quite a few unwraps, I think it'll need a bigger refactoring for proper error management.